test/position-managers/Execute.t.sol

```diff
@@ -0,0 +1,170 @@
+        uint256 initialiLiquidity,
+        uint256 liquidityToAdd,
+        uint256 liquidityToAdd2
+    ) public {
+        initialiLiquidity = bound(initialiLiquidity, 1e18, 1000e18);
+        liquidityToAdd = bound(liquidityToAdd, 1e18, 1000e18);
+        liquidityToAdd2 = bound(liquidityToAdd2, 1e18, 1000e18);
+        mint(range, initialiLiquidity, address(this), ZERO_BYTES);
+        uint256 tokenId = lpm.nextTokenId() - 1;
+
+        Planner.Plan memory planner = Planner.init();
+
+        planner = planner.add(Actions.INCREASE, abi.encode(tokenId, liquidityToAdd, ZERO_BYTES));
+        planner = planner.add(Actions.INCREASE, abi.encode(tokenId, liquidityToAdd2, ZERO_BYTES));
+
+        bytes memory calls = planner.finalize(range.poolKey);
+        lpm.modifyLiquidities(calls, _deadline);
+
+        bytes32 positionId =
+            keccak256(abi.encodePacked(address(lpm), range.tickLower, range.tickUpper, bytes32(tokenId)));
+        (uint256 liquidity,,) = manager.getPositionInfo(range.poolKey.toId(), positionId);
+
+        assertEq(liquidity, initialiLiquidity + liquidityToAdd + liquidityToAdd2);
+    }
+
+    // this case doesnt make sense in real world usage, so it doesnt have a cool name. but its a good test case
+    function test_execute_mintAndIncrease(uint256 initialLiquidity, uint256 liquidityToAdd) public {
+        initialLiquidity = bound(initialLiquidity, 1e18, 1000e18);
+        liquidityToAdd = bound(liquidityToAdd, 1e18, 1000e18);
+
+        uint256 tokenId = 1; // assume that the .mint() produces tokenId=1, to be used in increaseLiquidity
+
+        Planner.Plan memory planner = Planner.init();
+
+        planner = planner.add(Actions.MINT, abi.encode(range, initialLiquidity, address(this), ZERO_BYTES));
+        planner = planner.add(Actions.INCREASE, abi.encode(tokenId, liquidityToAdd, ZERO_BYTES));
+
+        bytes memory calls = planner.finalize(range.poolKey);
+        lpm.modifyLiquidities(calls, _deadline);
+
+        bytes32 positionId =
+            keccak256(abi.encodePacked(address(lpm), range.tickLower, range.tickUpper, bytes32(tokenId)));
+        (uint256 liquidity,,) = manager.getPositionInfo(range.poolKey.toId(), positionId);
+
+        assertEq(liquidity, initialLiquidity + liquidityToAdd);
+    }
+
+    // rebalance: burn and mint
+    function test_execute_rebalance() public {}
+    // coalesce: burn and increase
+    function test_execute_coalesce() public {}
+    // split: decrease and mint
+    function test_execute_split() public {}
+    // shift: decrease and increase
+    function test_execute_shift() public {}
+    // shard: collect and mint
+    function test_execute_shard() public {}
+    // feed: collect and increase
+    function test_execute_feed() public {}
+
+    // transplant: burn and mint on different keys
+    function test_execute_transplant() public {}
+    // cross-coalesce: burn and increase on different keys
+    function test_execute_crossCoalesce() public {}
```

Comment:
lmao i like the fun name ideas but i do think it would be much easier to see what was failing if you saw
```
test_execute_burnAndIncreaseDifferentKeys
```

instead of `test_execute_crossCoalesce`

Reply:
I am gonna keep this for now - sauce has a really great doc/table explaining all these names LOL. we can change later when we actually implement the tests if u still dont like